src/store/slices/projectSlice.ts

```diff
@@ -109,6 +112,12 @@ export const projectSlice = createSlice({
         state.images[index].categoryId = action.payload.categoryId;
       }
     },
+    updateTileSize(
+      state: Project,
+      action: PayloadAction<{ newValue: number }>
+    ) {
+      state.tileSize = action.payload.newValue!;
```

Comment:
You can remove the `!` since the value should always be defined.

Reply:
The trouble I had was that the slider widget event seems to be typed to return `number | number[]`. I think the latter happens when you have a range slider rather than a single one, but I couldn't figure out how to tell it not to worry about the possible array type?